src/many/src/message.rs

```diff
@@ -181,24 +186,131 @@ impl CoseSign1RequestMessage {
         }
     }
 
+    /// Perform WebAuthn request verification
+    ///
+    /// This is non-standard COSE
+    /// See https://webauthn.guide/#webauthn-api
+    fn _verify_webauthn(
+        &self,
+        unprotected: BTreeMap<Label, Value>,
+        key: CoseKeyIdentity,
+    ) -> Result<(), String> {
+        #[derive(Deserialize)]
+        struct ClientData {
```

Comment:
Can't you get this type from a crate? Seems like the webauthn-rs crate could be used; https://docs.rs/webauthn-rs/latest/webauthn_rs/proto/struct.CollectedClientData.html

Reply:
I can. However, is it really worth it to bring in this crate (only) for this?